src/proto/grpc/extension/extension.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package grpc.extension;
+
+message ClientConfig {
```

Comment:
ExtensionConfig ?

Reply:
Prefer to keep ClientConfig, as currently all the features are on the client side.

Reply:
As discussed, ExtensionConfig is much less confusion.